tools/speech_data_simulator/README.md

```diff
@@ -0,0 +1,112 @@
+**Speech Data Simulator**
+===============
+
+Outline
+------------
+
+The speech data simulator generates synthetic multispeaker audio sessions for training or evaluating models for multispeaker ASR or speaker diarization. This tool aims to address the lack of labelled multispeaker training data and to help models deal with overlapping speech.
+
+The simulator loads audio files from different speakers as well as forced alignments for each sentence and concatenates the audio files together to build a synthetic multispeaker audio session. The simulator uses the word alignments to segment the audio from each speaker to produce utterances of the desired length. The simulator also incorporates synthetic room impulse response generation in order to simulate multi-microphone multispeaker sessions.
+
+Features
+------------
+
+The simulator is reconfigurable and has several options including:
```

Comment:
Please add one-linear explanation about these arguments(parameters). It is quite hard to understand what are these parameters:

Example:
 Speaker dominance:
      Determines how much portion will be taken by a speaker in a session. The higher, the one speaker takes up more time in one session. etc...